src/components/Input/index.cy.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import Input, { inputTestIds } from './';
+
+describe('Input', () => {
+  it('renders the input with the correct attributes', () => {
+    cy.mount(<Input name="first_name" type="text" />);
+
+    cy.findByTestId(inputTestIds.input)
+      .should('have.attr', 'id', 'firstName')
+      .and('have.attr', 'name', 'first_name')
+      .and('have.attr', 'type', 'text');
+  });
+
+  it('renders the label if label attribute is set', () => {
+    cy.mount(<Input name="first_name" type="text" label="First name" />);
+
+    cy.findByTestId(inputTestIds.label).should('be.visible').and('have.text', 'First name').and('have.attr', 'for', 'firstName');
```

Comment:
```suggestion
    cy.findByTestId(inputTestIds.label)
      .should('be.visible')
      .and('have.text', 'First name')
      .and('have.attr', 'for', 'firstName');
```
Should we break down the statement here?

Reply:
The prettier warns when I put it on a new line.

> 18:40  error  Replace `⏎······.should('be.visible')⏎······.and('have.text',·'First·name')⏎······` with `.should('be.visible').and('have.text',·'First·name')`  prettier/prettier

Seems it is still an issue on Prettier side 🤕

Reply:
After updated tests to nested blocks, the line now exceeds the printWidth and it forces to use new line 😆